Library/Formula/mosquitto.rb

```diff
@@ -14,12 +14,13 @@ class Mosquitto < Formula
   depends_on "pkg-config" => :build
   depends_on "cmake" => :build
   depends_on "c-ares"
+  depends_on "libwebsockets"
 
   # mosquitto requires OpenSSL >=1.0 for TLS support
   depends_on "openssl"
 
   def install
-    system "cmake", ".", *std_cmake_args
+    system "cmake", "-DWITH_WEBSOCKETS=ON", ".", *std_cmake_args
```

Comment:
The support for websockets in mosquitto is very useful in my opinion and its arguably a core feature of version 1.4. So +1 for recommended instead of optional if it should be configurable.